### ISSUE TITLE
KeyError: 'row0'

### ISSUE THREAD

**Issue by dbrgn** [codywd/NetGUI#42]
When starting NetGUI for the first time:

```
Please wait! Now Scanning.
Done Scanning!
/tmp/essid.AMNb7Xsw
\x00
Traceback (most recent call last):
  File "/usr/share/netgui/main.py", line 619, in <module>
    netgui()
  File "/usr/share/netgui/main.py", line 78, in __init__
    self.InitUI()
  File "/usr/share/netgui/main.py", line 182, in InitUI
    self.startScan(None)
  File "/usr/share/netgui/main.py", line 230, in startScan
    self.checkScan()
  File "/usr/share/netgui/main.py", line 274, in checkScan
    self.APStore.set(aps["row" + str(i)], 1, percent)
KeyError: 'row0'
```

**Comment by bich30** [comment]
I have the same problem.

Attach the output file:

[root@tmp]# cat /tmp/essid.3BGi7rgZ 
-55	[WPA2-PSK-CCMP][ESS]	\x00
-65	[WPA2-PSK-CCMP][ESS]	ElC
-65	[WPA2-PSK-CCMP][ESS]	Gxxx
-70	[WPA-PSK-CCMP+TKIP][ESS]	TECNOINF
-79	[WPA2-PSK-CCMP][ESS]	Bl00nde
-81	[WPA-PSK-CCMP+TKIP][WPS][ESS]	MOVISTAR_CXXX
-81	[WPA2-PSK-CCMP][ESS]	Mac mini de JJD
-81	[WPA-PSK-CCMP+TKIP][WPA2-PSK-CCMP+TKIP][ESS]	IF

**Comment by codywd** [comment]
I want to say this is resolved in the upcoming 0.85. I haven't seen it. I will close it for now with this, and if it happens again we'll  have to do more troubleshooting.
